dotnet-project-file-analyzers.sln

```diff
@@ -52,6 +52,7 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "C#", "C#", "{BAD262BB-6053-
 		projects\GeneratePackageNotPackable\GeneratePackageNotPackable.csproj = projects\GeneratePackageNotPackable\GeneratePackageNotPackable.csproj
 		projects\GeneratePackageOnBuildUnconditionally\GeneratePackageOnBuildUnconditionally.csproj = projects\GeneratePackageOnBuildUnconditionally\GeneratePackageOnBuildUnconditionally.csproj
 		projects\LegacyProject\LegacyProject.csproj = projects\LegacyProject\LegacyProject.csproj
+		projects\MaliciousIndenting\MaliciousIndenting.csproj = projects\MaliciousIndenting\MaliciousIndenting.csproj
```

Comment:
Malicious? Its a bit strong

Reply:
Well, do you have a better suggestion?

Reply:
Faulty, improper, noncompliant or something like illegible.

Reply:
Malicious feels like someone who is intent on harming with their indentation

Reply:
I went for Faulty. Thanks for the suggestion.